crates/sqlbuiltins/src/functions/scalars/df_scalars.rs

```diff
@@ -1,15 +1,4 @@
-// we make use of the document! macro to generate the documentation for the builtin functions.
-// specifically the `stringify!` macro is used to get the name of the function.
-// `Abs` would otherwise be `Abs` instead of `abs`. and so on.
-#![allow(non_camel_case_types)]
-
-use crate::{
-    document,
-    functions::{BuiltinFunction, ConstBuiltinFunction},
-};
-use datafusion::logical_expr::BuiltinScalarFunction;
-use protogen::metastore::types::catalog::FunctionType;
-
+use super::*;
```

Comment:
i think it's just a matter of personal preference. Since everything in the module has nearly identical imports, it seems logical to just do it once instead of in every submodule.

Reply:
Star imports definitely reduce readability for me. if we have a bunch of modules with identical imports that are all going to be exported in the same namespace, it sort of seems like we should just keep them all in one file. 

I have, however realized that this basically makes this package have the same semantics as Go packages 😂